packages/cli/package.json

```diff
@@ -170,7 +171,7 @@
     "ts-node": "10.9.1",
     "utility-types": "2.1.0",
     "vite": "5.1.6",
-    "vitest": "2.0.1",
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏As a follow-up task, can you update the vite version throughout the repo?